packages/code-du-travail-frontend/pages/index.js

```diff
@@ -27,12 +28,18 @@ const Home = ({ pageUrl, ogImage, children = [] }) => (
 );
 
 Home.getInitialProps = async () => {
-  const response = await fetch(`${API_URL}/themes`);
-  if (!response.ok) {
-    return { statusCode: response.status };
+  try {
```

Comment:
should this be done in _app ?

Reply:
the thing is we dont have the same props shape